lib/kadai-core/src/main/java/io/kadai/common/api/SharedConstants.java

```diff
@@ -19,5 +19,9 @@
 package io.kadai.common.api;
 
 public class SharedConstants {
-  public static String MASTER_DOMAIN = "";
+
+  private SharedConstants() {
```

Comment:
Why did you add this constructor?

Reply:
Because the class clearly must not be instantiated, and an empty private constructor overwrites the default one, which is public.